components/forms/List.vue

```diff
@@ -0,0 +1,91 @@
+<script setup lang="ts">
+const { formTableColumns } = useForms();
+defineProps({
+  forms: Array as PropType<Form[]>,
+});
+</script>
+
+<template>
+  <div class="form-list">
+    <table class="form-table">
+      <thead class="table-head">
+        <tr class="row">
+          <th
+            class="cell"
+            :class="{ '-action': value === 'action' }"
+            v-for="{ value, label } in formTableColumns"
+            :key="value"
+          >
+            {{ label }}
+          </th>
```

Comment:
chỗ này em để set variant luôn cho mỗi column nhé, nên mình sẽ không cần value === 'action' nữa, lấy luôn variant cho mỗi column dựa trên giá trị `value` của nó
```suggestion
          <th
            class="cell"
            :class="{ `-${value}` : true }"
            v-for="{ value, label } in formTableColumns"
            :key="value"
          >
            {{ label }}
          </th>
```